src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,64 @@
 import { useEffect, useState, useContext } from "react";
-import { Route, Routes, Link } from "react-router-dom";
+import { Route, Routes, Link, useParams } from "react-router-dom";
 import axios from "axios";
-import { getAllArticles } from "../utils";
+import { getAllArticles, getArticleById } from "../utils";
+import { getCommentsByArticleId } from "../utils";
 
-const ArticleCard = ({}) => {
-   
-}
+const ArticleCard = () => {
+  const { article_id } = useParams();
+  const [articlePage, setArticlePage] = useState([]);
+  const [comments, setComments] = useState([]);
```

Comment:
Some loading state for visual feedback for the user would be very much included whilst a comment is fetched